src/reference/asciidoc/amqp.adoc

```diff
@@ -1059,11 +1059,14 @@ public IntegrationFlow pubSubInFlow(ConnectionFactory connectionFactory) {
 === AMQP Message Headers
 
 The Spring Integration AMQP Adapters will map standard AMQP properties automatically.
-These properties will be copied by default to and from Spring Integration `MessageHeaders` using the http://static.springsource.org/spring-integration/api/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.html[DefaultAmqpHeaderMapper].
+These properties will be copied by default to and from Spring Integration `MessageHeaders` using the
+http://docs.spring.io/spring-integration/api/org/springframework/integration/amqp/support/DefaultAmqpHeaderMapper.html[DefaultAmqpHeaderMapper].
```

Comment:
Yeah... After moving to the `SimpleAmqpHeaderMapper` we may lose the `negate` functionality.

Although, from other side, I don't see the use utilize its support for the HTTP, for example.
Or for any other `HeaderMapper` implementation...
WDYT ?